cli/lib/kontena/cli/apps/common.rb

```diff
@@ -90,6 +90,8 @@ def create_yml(services, file='kontena.yml')
     def app_json
       if !@app_json && File.exist?('app.json')
         @app_json = JSON.parse(File.read('app.json'))
+      else
+        @app_json = {}
```

Comment:
Is it ok to initialize `@app_json` to empty hash on every call (if file is missing)?